docassemble/ALToolbox/al_income.py

```diff
@@ -555,36 +627,41 @@ def total(self, source=None):
 
 class ALItemizedValue(DAObject):
     """
-    An item in an ALItemizedValueDict (a line item like wages or union dues).
+    An item in an ALItemizedValueDict (a line item like wages, tips or union dues).
     Should be a positive number, even if it will later be subtracted from the
     job's net total.
 
-    WARNING:
-    This item's period-based value can't be calculated correctly outside of an
-    ALItemizedJob. Its value should only be accessed through the filtering
-    methods of the ALItemizedJob that contains it.
+    WARNING: This item's period-based value can't be calculated correctly
+    outside of an ALItemizedJob. Its value should only be accessed through the
+    filtering methods of the ALItemizedJob that contains it.
 
     Attributes:
-    .value {num} A number representing an amount of money accumulated during
-        the `times_per_year` of this item or this item's job.
+    .value {float | Decimal} A number representing an amount of money accumulated
+        during the `times_per_year` of this item or this item's job.
     .is_hourly {bool} Whether this particular item is calculated hourly.
-    .times_per_year {str | num} A number (or string representing a number)
-        representing the annual frequency of the job. E.g. 12 for monthly.
+    .times_per_year { float} A denominator of a year representing the annual
+         frequency of the job. E.g. 12 for monthly.
+    .exists {bool} (Optional) Allows an interview author to pre-define some common
+        descriptors, like "wages" or "union dues" without requiring the user to
+        provide a value for each item.
+
+        If the ".exists" attribute is False or undefined, the item will not be used
+        when calculating totals.
```

Comment:
I think I may have just forgotten, but I don't clearly remember working with `.exist` for this class. Maybe it's a docassemble thing. Either way, I just wanted to let you know and am willing to go with whatever makes sense.

Reply:
I think this is also behavior we can adjust after trying on @CaroRob's form